front/components/assistant/conversation/Conversation.tsx

```diff
@@ -0,0 +1,53 @@
+import { useConversation } from "@app/lib/swr";
+
+export default function Conversation({
+  conversationId,
+  workspaceId,
+}: {
+  conversationId: string;
+  workspaceId: string;
+}) {
+  const { conversation, isConversationError, isConversationLoading } =
+    useConversation({
```

Comment:
Would be nice to get the mutate out of the useConversation we have previous art about that in swr.ts so that we don't rely on brittle mutating by URL

Reply:
The mutate is now being returned, I will use it in my next PR.